woocommerce/woocommerce-admin-test-helper
# Add Experiments -- helps to toggle experiment variation

This PR adds the Experiments tab. It allows people without Automattic VPN to assign themselves to treatment group of an experiment.
![Screen Shot 2022-02-17 at 12 35 33 PM](https://user-images.githubusercontent.com/4723145/154566675-31464074-cc9c-4a6b-9018-be9901e8e8ea.jpg)

### Test Instructions

1. Checkout this branch and run `npm start`
2. Navigate to Tools -> WCA Test Helper
3. Click `Experiments`
4. Click `Toggle` button on one of the experiments.
5. Open your browser insepctor -> Application -> Local Storage
6. Confirm the experiment variation has been updated.
